benchexec/tools/symbiotic4.py

```diff
@@ -46,24 +46,30 @@ def name(self):
         """
         return "symbiotic"
 
-    def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
+    def cmdline(self, executable, options, task, rlimits):
         """
         Compose the command line to execute from the name of the executable
         """
 
-        if propertyfile is not None:
-            options = options + ["--prp={0}".format(propertyfile)]
+        if task.property_file is not None:
+            options = options + ["--prp={0}".format(task.property_file)]
 
-        return [executable] + options + tasks
+        data_model_param = get_data_model_from_task(
+            task, {"ILP32": "--32", "LP64": "--64"}
+        )
+        if data_model_param and data_model_param not in options:
+            options += [data_model_param]
 
-    def determine_result(self, returncode, returnsignal, output, isTimeout):
-        if isTimeout:
+        return [executable] + options + list(task.input_files_or_identifier)
+
+    def determine_result(self, run):
+        if run.was_timeout:
             return "timeout"
 
-        if output is None:
+        if run.output is None:
```

Comment:
This is never satisfied (same holds true for condition in previous code, actually).

Reply:
fixed in 78f718139b1300c094474898e74de1173ee09417